packages/mantine/src/components/pages/auth/components/login/index.tsx

```diff
@@ -44,26 +38,30 @@ export const LoginPage: React.FC<LoginProps> = ({
     renderContent,
     formProps,
 }) => {
+    const { useForm, FormProvider } = FormContext;
+    const { onSubmit: onSubmitProp, ...useFormProps } = formProps || {};
     const translate = useTranslate();
     const { Link } = useRouterContext();
 
-    const { getInputProps, onSubmit } = useForm({
+    const form = useForm({
         initialValues: {
             email: "",
             password: "",
             remember: false,
         },
         validate: {
-            email: (value) =>
+            email: (value: any) =>
```

Comment:
```suggestion
            email: (value: any) =>
```
could we use strict type? @salihozdemir @yildirayunlu

Reply:
`FormType` interface can be passed as `createFormContext` generic type. However, we cannot do this in this use. 

We can come up with a different solution to this later.